target/linux/ath79/dts/qca9563_tplink_tl-wpa8630p-v2.dtsi

```diff
@@ -0,0 +1,129 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
```

Comment:
dts-v1 should only be in DTS, remove it here.

Reply:
Good to know, I was actually asking myself where it should be defined!

Reply:
Long story: dts-v1 is only valid when given once at the beginning of the file.
Since dtsi files are simply included as additional text, technically it would not matter whether you specify dts-v1 in DTS _or_ DTSI.
If you specify in both, you would get two entries in the resulting file after includes have been resolved, which would be violating the specifications. However, parser would just ignore the second one, and nothing would happen anyway.

Based on that, I personally request the dts-v1 in DTS files, as this makes it easy to keep track of it and to prevent multiple inclusion in case they ever change behavior of that parser.

Reply:
Thanks for the explanation! 
This is what happens if people (=me) don't read specifications and just hack stuff together while only partially understanding the background.

Reply:
Actually, this is not written down anywhere, and I had to ask on the devicetree-spec mailing list to get a substantial answer :-)